topostats/entry_point.py

```diff
@@ -854,6 +861,14 @@ def create_parser() -> arg.ArgumentParser:
         required=False,
         help="Extract height profiles along maximum feret of molecules",
     )
+    process_parser.add_argument(
```

Comment:
This is wrong the argument should be added to the `grainstats_parser` and not `process_parser` (the hazards of copy and pasting!).

```suggestion
    grainstats_parser.add_argument(
```